shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-sqlserver/src/main/antlr4/imports/sqlserver/DMLStatement.g4

```diff
@@ -57,7 +57,7 @@ assignmentValue
     ;
 
 delete
-    : DELETE (singleTableClause | multipleTablesClause) whereClause?
+    : withClause_? DELETE top? (singleTableClause | multipleTablesClause) outputClause_? whereClause?
```

Comment:
@dongzl plz change `withClause_` to `withClause`,`outputClause_` to `outputClause`.